### DISCUSSION TITLE
Arithmetic functions

### DISCUSSION THREAD

**Discussion by rbalagangadharan** [sqlkata/querybuilder#710] [Q&A] [answered]
Do we have support for below's

- Length
- Ceiling
- Floor
- Roundx

e.g 

1. SELECT * FROM CUSTOMER WHERE LENGTH(CUSTOMER_NAME) = 10
2. SELECT * FROM CUSTOMER WHERE ROUND(SALARY) = 100
3. SELECT * FROM CUSTOMER WHERE CEILING(SALARY) = 100
4. SELECT * FROM CUSTOMER WHERE FLOOR(SALARY) = 100

**Comment by ahmad-moussawi** [comment, marked as answer]
```cs
db.Query("CUSTOMER").WhereRaw("LENGTH(CUSTOMER_NAME) = ?", new {} [10]);
```
Check `https://sqlkata.com/docs/where#where-raw`
